### DISCUSSION TITLE
Read SSH key from a subkey

### DISCUSSION THREAD

**Discussion by ivan770** [openpgpjs/openpgpjs#1295] [Q&A] [answered]
Is it possible to read SSH key derived from a subkey with `authentication` flag?
Certainly, there are utilities for CLI (`gpg` supports this via `export-ssh-key` flag), however, I'm interested in using JS, without any calls to system utilities.

**Comment by twiss** [comment, marked as answer]
There is no functionality for this at the moment, sorry.
